### DISCUSSION TITLE
Logging using fastapi CLI

### DISCUSSION THREAD

**Discussion by liquidcarbon** [fastapi/fastapi#13034] [Questions]
### First Check

- [X] I added a very descriptive title here.
- [X] I used the GitHub search to find a similar question and didn't find it.
- [X] I searched the FastAPI documentation, with the integrated search.
- [X] I already searched in Google "How to X in FastAPI" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/pydantic/pydantic).
- [X] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
- [X] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
Using uvicorn, one could customize the logger via `uvicorn.config.LOGGING_CONFIG` prior to `uvicorn.run`.
```

### Description

Is there a recipe for setting up logging using new `fastapi run`/`fastapi dev` CLI?

### Operating System

Linux

### FastAPI Version

0.115.6

### Pydantic Version

2.10.3

### Python Version

3.12

**Comment by pmareke** [comment]
I had the same problem.

After a few hours I realised that the problem was with the `fastapi run` command, if I use the `uvicorn` command the logs appears automatically.

Thanks @liquidcarbon for open the discussion!

**Comment by liquidcarbon** [comment]
Perhaps `fastapi run` could use a `--logging-format` and `--logging-access-format` options,
or a subcommand `fastapi logging` to setup logging for a given project to keep `fastapi run` simple and elegant as it is.

**Reply by mordilos** [reply]
I second that, keeping it all together using FastAPI cli  is very useful!

**Comment by YuriiMotov** [comment]
It's more related to FastAPI CLI and there is a discussion in that repo: https://github.com/fastapi/fastapi-cli/discussions/13

Let's track it there
